VERSION

```diff
@@ -1 +1 @@
-1.7.2
```

Comment:
We want to update the README as well, as for previous rc https://github.com/kubernetes/kube-state-metrics/pull/814/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
done